meta-jsdelivr/recipes-jsdelivr/jsdelivr-scripts/files/jsdelivr-updateContainerAuto.sh

```diff
@@ -0,0 +1,113 @@
+#!/bin/bash
+
+echo "JSDELIV AUTO Update start" > /dev/tty5
+
+echo "STOPING the MANDATORY reboot script" > /dev/tty5
+killall -STOP jsdelivr-mandatoryReboot.sh
+killall -STOP jsdelivr-systemMonitor.sh
+
+
+killall  -9 jsdelivr-systemWatchdog.sh
+sleep 1
+/usr/bin/jsdelivr-keepWatchdogHappy.sh &
+
+docker stop $(docker ps -a -q)
+/bin/systemctl stop containerd
+/bin/systemctl stop docker
+umount /var/lib/docker/overlay2/*/*
+umount /var/lib/docker
+echo 1 > /sys/block/zram0/reset
+
+
+mkdir /tmp/AutoUpdate
+mount -o ro /dev/mmcblk0p4 /tmp/AutoUpdate
+
+if mount | grep "/tmp/AutoUpdate" > /dev/null; then
+    echo "Partition mounted" > /dev/tty5
+else
+    echo "Unable to mount autoupdate partition, Aborting update" > /dev/tty5
+    echo "Resuming the MANDATORY reboot script" > /dev/tty5
+    killall -CONT jsdelivr-mandatoryReboot.sh
+    killall jsdelivr-keepWatchdogHappy.sh
+    exit 1
+fi
+
+SERVER_VER=`curl --silent https://api.github.com/repos/jsdelivr/globalping-probe/releases/latest | awk -F ':'  '/"tag_name"/ {print  substr($NF, 1, length($NF)-1)}'`
```

Comment:
Please make sure we don't query GitHub services anywhere, as they don't support IPv6. Here, you can use this URL, which always returns the latest version: https://data.jsdelivr.com/v1/packages/gh/jsdelivr/globalping-probe/resolved (if we need anything else from GitHub APIs, let me know).

Reply:
Change done